curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/introducing-else-statements.english.md

```diff
@@ -42,7 +42,7 @@ tests:
     testString: assert(testElse(6) === "Bigger than 5");
   - text: <code>testElse(10)</code> should return "Bigger than 5"
```

Comment:
```suggestion
  - text: <code>testElse(10)</code> should return "Bigger than 5".
```

Reply:
@moT01 Can not commit suggested change "as-is" because you have deleted a space off the beginning of the line which must be there.